src/widgets/school-menu/ui/school-item/school-item.tsx

```diff
@@ -1,6 +1,8 @@
 import { Link } from 'react-router-dom';
 import { GenericItemProps } from '../school-list/school-list';
+import { COURSE_STALE_AFTER_DAYS } from '@/app/const';
 import { type Course } from '@/app/types';
+import { getCourseDate } from '@/shared/helpers/getCourseDate';
```

Comment:
As I understood we need 'helpers/index.ts' file to export helpers functions.

Reply:
Are this changes relevant to this task?